pkg/handler/eventhandler.go

```diff
@@ -117,7 +123,7 @@ func isPriorityQueue[request comparable](q workqueue.TypedRateLimitingInterface[
 // Create implements EventHandler.
 func (h TypedFuncs[object, request]) Create(ctx context.Context, e event.TypedCreateEvent[object], q workqueue.TypedRateLimitingInterface[request]) {
```

Comment:
Doing or caching this directly in the `Funcs` is problematic:
* It doesn't have a constructor, so can't do it there (We have to remember to never ever export types and only constructors for this kind of scenario)
* It has value receivers and we need some kind of locking if we want to cache this, but using a lock with value receivers means we copy the lock which the linter forbids